### ISSUE TITLE
bak-ContosoV2-100M.bak.7z. with the extensions .001 and so on are corrupt files

### ISSUE THREAD

**Issue by HarshJoshi1997** [sql-bi/Contoso-Data-Generator-V2-Data#1]
Hello There,

I downloaded the files with the extensions 7z.001 and so on and tried to extract it using both the tools i.e., 7-Zip and WinRAR and in both the cases, it's giving an error stating that the files are corrupt and cannot be extracted.

Can this be fixed and re-uploaded?

Thanks,

**Comment by marcosqlbi** [comment]
When the files has a number (like .001) you need to download all the files with consecutive numbers and use 7-Zip do unzip them.
Make sure you use the latest version of 7-Zip.
Let me know if this fixes the problem - if not, please copy the exact error message from 7-Zip, thanks!

**Comment by DinaW21** [comment]
Ho provato anche io ad effettuare il download e ad estrarre con 7-zip ma mi dice che non è possibile aprire il file come archivio. 
Sto sbagliando in qualcosa? Grazie mille.

**Comment by marcosqlbi** [comment]
@DinaW21  @HarshJoshi1997 
Please, try the following:

1. Download in the same folder **all the files** .001 .. .012 (or whatever the last number available is for the file you need
2. Open 7-Zip and point to the folder where you downloaded the files at (1)
3. Select the file ".001" and click "Extract"
4. Complete the extraction

I made a complete test, downloading all the files for bak-ContosoV2-100M.bak.0xx and following the instructions above - it works.
If you face an error, please report the complete error message and a screenshot of the problem.

Thanks!

**Comment by DinaW21** [comment]
I tried to follow all the steps indicated but it always gives me the same error. I am attaching the photo.
![IMG_4123](https://github.com/user-attachments/assets/5d5b5a76-4346-47e2-99a0-34ed108129da)

**Comment by marcosqlbi** [comment]
It's possible that the file is still open - perhaps, do you have an antivirus that is checking the file while 7-Zip tries to extract it?
Also check the version of 7-Zip, this is the one I use.
![image](https://github.com/user-attachments/assets/3808f13e-499f-4248-aae8-aa4eda9be8be)

**Comment by DinaW21** [comment]
No, non ho antivirus e la versione di 7 zip corrisponde. Continua a darmi lo stesso errore. Ma devo scaricare tutti i file o basta soltanto bak-ContosoV2-100M.bak.7z.001<https://github.com/sql-bi/Contoso-Data-Generator-V2-Data/releases/download/ready-to-use-data/bak-ContosoV2-100M.bak.7z.001>?

Grazie ancora!
________________________________
Da: Marco Russo ***@***.***>
Inviato: mercoledì 31 luglio 2024 06:08
A: sql-bi/Contoso-Data-Generator-V2-Data ***@***.***>
Cc: Dina ***@***.***>; Mention ***@***.***>
Oggetto: Re: [sql-bi/Contoso-Data-Generator-V2-Data] bak-ContosoV2-100M.bak.7z. with the extensions .001 and so on are corrupt files (Issue #1)


It's possible that the file is still open - perhaps, do you have an antivirus that is checking the file while 7-Zip tries to extract it?
Also check the version of 7-Zip, this is the one I use.
image.png (view on web)<https://github.com/user-attachments/assets/3808f13e-499f-4248-aae8-aa4eda9be8be>

**Comment by marcosqlbi** [comment]
@DinaW21 you should download **all** the files .0xx

**Comment by usrbinr** [comment]
this worked for me
